packages/quantic/force-app/main/default/lwc/quanticTab/__tests__/quanticTab.test.js

```diff
@@ -158,24 +158,50 @@ describe('c-quantic-tab', () => {
       );
     });
 
-    it('should pass the clearFiltersOnTabChange option to the tab controller when set to true', async () => {
-      const optionsWithClearFilters = {
-        ...defaultOptions,
-        clearFiltersOnTabChange: true,
-      };
-      createTestComponent(optionsWithClearFilters);
-      await flushPromises();
-
-      expect(functionsMocks.buildTab).toHaveBeenCalledTimes(1);
-      expect(functionsMocks.buildTab).toHaveBeenCalledWith(exampleEngine, {
-        options: {
-          expression: optionsWithClearFilters.expression,
-          id: optionsWithClearFilters.name,
+    describe('the clearFiltersOnTabChange property', () => {
+      it('should pass the clearFiltersOnTabChange option to the tab controller when set to true', async () => {
+        const optionsWithClearFilters = {
+          ...defaultOptions,
           clearFiltersOnTabChange: true,
-        },
-        initialState: {
-          isActive: optionsWithClearFilters.isActive,
-        },
+        };
+        createTestComponent(optionsWithClearFilters);
+        await flushPromises();
+
+        expect(functionsMocks.buildTab).toHaveBeenCalledTimes(1);
+        expect(functionsMocks.buildTab).toHaveBeenCalledWith(exampleEngine, {
+          options: {
+            expression: optionsWithClearFilters.expression,
+            id: optionsWithClearFilters.name,
+            clearFiltersOnTabChange: true,
+          },
+          initialState: {
+            isActive: optionsWithClearFilters.isActive,
+          },
+        });
+      });
+
+      ['true', 'false'].forEach((value) => {
+        it(`should pass the clearFiltersOnTabChange option as a boolean to the tab controller when a ${value} string is provided`, async () => {
+          const expectedClearFiltersValue = value === 'true' ? true : false;
+          const optionsWithClearFilters = {
+            ...defaultOptions,
+            clearFiltersOnTabChange: value,
+          };
+          createTestComponent(optionsWithClearFilters);
+          await flushPromises();
+
+          expect(functionsMocks.buildTab).toHaveBeenCalledTimes(1);
+          expect(functionsMocks.buildTab).toHaveBeenCalledWith(exampleEngine, {
+            options: {
+              expression: optionsWithClearFilters.expression,
+              id: optionsWithClearFilters.name,
+              clearFiltersOnTabChange: expectedClearFiltersValue,
+            },
+            initialState: {
+              isActive: optionsWithClearFilters.isActive,
+            },
+          });
+        });
```

Comment:
I don't think we want to get into all this to parse a boolean out of a value that will **always** be a string.

And the standard is that if the value is set on the element, it's always supposed to be truthy.